tests/xtimer_cli/tests/Xtimer.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (C) 2018 Kevin Weiss <kevin.weiss@haw-hamburg.de>
```

Comment:
Ahh this is what thomas was talking about, please change to Copyright (C) 2018 Kevin Weiss, for HAW <kevin.weiss@haw-hamburg.de> (or remove completely).